content/docs/05.concepts/expression/04.function.md

```diff
@@ -568,4 +610,15 @@ The above example will:
 1. retrieve the secret `GITHUB_ACCESS_TOKEN` from an environment variable `SECRET_GITHUB_ACCESS_TOKEN`
 2. base64-decode it at runtime.
 
+## yaml
+
+The `yaml` function will convert any string to an object allowing to access its properties.
+
+```twig
+"{{ yaml('foo: [666, 1, 2]').foo[0] }}"
+{# results in: '666' #}
+
+{{ yaml(yaml_object) | jq(...) | yaml }}
```

Comment:
```suggestion
"{{ yaml(yaml_object) | jq(...) | yaml }}"
```

Reply:
Can you actually give this example some dummy data so it works? replace `yaml_object` and the `...` in `jq`.

Reply:
@shrutimantri tagging for visibility